scripts/export_demucs_onnx.py

```
@@ -265,97 +443,60 @@ def _real_valued_stft(
+smoke_in   = np.zeros((1, 1, TRAIN_LEN), dtype=np.float32)
+outputs    = sess.run(None, {'input': smoke_in})
 out        = outputs[0]
-expected_T = SAMPLE_RATE * 3
 
-assert out.shape == (1, 1, expected_T), \
-    f'[FAIL] shape {out.shape} != (1, 1, {expected_T})'
-assert out.min() >= 0.0,         f'[FAIL] min={out.min()} < 0'
-assert out.max() <= 1.0 + 1e-4, f'[FAIL] max={out.max()} > 1'
-print(f'[8/8] Smoke test PASSED  shape={list(out.shape)}  '
+assert out.shape[0] == 1 and out.shape[1] == 1 and out.shape[2] == TRAIN_LEN, \
+    f'[FAIL] shape {out.shape}'
```

Ruff (0.15.12)

[warning] Assertion should be broken down into multiple parts

(PT018)